src/main/java/se/sundsvall/document/api/model/DocumentUpdateRequest.java

```diff
@@ -24,6 +27,9 @@ public class DocumentUpdateRequest {
 	@Schema(description = "List of DocumentMetadata objects.")
 	private List<@Valid DocumentMetadata> metadataList;
 
+	@Schema(description = "Key is filename and value represents if a specific file should be archived.", requiredMode = NOT_REQUIRED)
+	private Map<String, Boolean> archiveMap;
```

Comment:
This attribute lives in the documentDataEntity. Why not use PATCH for documentData instead?
It would be more intuitive and you don't have to use a map,

Reply:
Using a separate request for updating a single files archive flag would most likely result in a lot of revisions, that might not matter but its part of the reasoning. This option gives us the possiblilty to update all files in a single request. I also don't see why it is an issue to use a map?

Reply:
I think that using a map that contains references to a different part of the object hierarchy in order to manipulate this data is not good API-design. Specially since there are RESTful alternatives. Don't you think?